### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: react-delegationdashboard/src/components/Trim/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const Trim = ({ text, dataTestId = '' }: TrimType) => {
+  const [debounce, setDebounce] = React.useState(0);
+  const [overflow, setOverflow] = React.useState(false);
+  const wrapperRef = React.useRef(document.createElement('span'));
+  const childRef = React.useRef(document.createElement('span'));
+
+  const debounceTracker = useDebounce(debounce, 100);
+
+  const listener = () => {
```
Comment (ccorcoveanu) [inline react-delegationdashboard/src/components/Trim/index.tsx:17]: If this is used only for text overflow there's a way to achieve this only with css, ping me if that's the case and you need a hint :)

**Review by ccorcoveanu — CHANGES_REQUESTED** [review]
Looks good 👍 

We should fix all the warnings for missing `;` inside the code

File: react-delegationdashboard/src/components/NodesList/InactiveNodeRow.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    <div className={`status-badge ${statusColor}`}>
+                        <div className="d-flex align-items-center">
+
+                            <span className={`led mr-1 bg-${statusColor}`} />
+                            {key.status.value}
+                        </div>
+                    </div>
+                ) : (
+                        <div className={`status-badge ${statusColor}`}>
+                            <div className="d-flex align-items-center">
```
[the remark this answers is not in the document]
Comment (danielailie) [reply react-delegationdashboard/src/components/NodesList/InactiveNodeRow.tsx:38]: I will use this once I will add the queue functionality

File: react-delegationdashboard/src/assets/styles/theme.scss [hunk 3]
```diff
@@ -13,9 +13,11 @@ Override Bootstrap variables only above.
 
 // Main Shards Dashboard
 @import './shards-dashboard/shards-dashboards';
-@import './../../components/StatCard/stats.scss';
```
Comment (ccorcoveanu) [inline react-delegationdashboard/src/assets/styles/theme.scss]: I would have only one `modules` folder, and `stats.scss`, `trim.scss` etc inside

**Review by ccorcoveanu — APPROVED** [review]

**Pull request merged**